<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-LEJP8C6" failures="1" tests="2" name="Test" time="17.906" errors="0" timestamp="2023-05-03T14:10:19 IST">
  <testcase classname="TestCases.VerifyLogin" name="InvalidLogin" time="0.734">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Error message is not valid expected [Please check your username. If you still can&amp;apos;t log in, contact your Salesforce administrator.] but found [Please check your username and password. If you still can&amp;apos;t log in, contact your Salesforce administrator.]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Error message is not valid expected [Please check your username. If you still can't log in, contact your Salesforce administrator.] but found [Please check your username and password. If you still can't log in, contact your Salesforce administrator.]
at TestCases.VerifyLogin.InvalidLogin(VerifyLogin.java:26)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- InvalidLogin -->
  <testcase classname="TestCases.VerifySignup" name="signup" time="8.516"/>
</testsuite> <!-- Test -->
